src/Form/EntryForm.php

```diff
@@ -51,13 +59,14 @@ public function getFormID() {
    */
   public function buildForm(array $form, FormStateInterface $form_state, AccountInterface $user = null) {
     // Check flood tables.
-    //@TODO Reimplement Flood Controls.
+
```

Comment:
I'm curious why you didn't implement the flood controls here -- they are commented out.